testbed/README.md

```diff
@@ -13,6 +13,12 @@ a global instance of `testbed.TestResultsSummary` to it.
 Each test case within the suite should create a `testbed.TestCase` and supply implementations of each of the various
 interfaces the `NewTestCase` function takes as parameters.
 
+`testbed.TestCase` uses `LoadGenerator` and `MockBackend` to further encapsulate pluggable components. `LoadGenerator` further encapsulates `DataProvider` and `DataSender` in order to generate and send  data.  `MockBackend` further encapsulate `DataReceiver` and provide comsume functionality.
```

Comment:
I wonder if it makes sense to put this in a `## Data Flow` section and talk about how data is generated by a `LoadGenerator` and consumed by a `MockBackend`, then point out how those two objects/structs wrap a dataProvider+sender and a receiver, respectively.

Reply:
Wow, sounds great! Let me modify it and make another commit, thank you!